modules/cpmso-openstack-ts-az-config.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * machine_management/control_plane_machine_management/cpmso-troubleshooting.adoc
+
+:_content-type: REFERENCE
+[id="cpmso-openstack-ts-az-config_{context}"]
+= Configuring {rh-openstack} Availability Zones type
+
+You have installed a cluster on {rh-openstack-first} or on existing {product-title} cluster version 4.13 or lower, and the root volumes are deployed without updating Availability Zones (AZ) in the `install-config.yaml` file. 
+
+This makes the root volume availability zone to be taken from the compute availability zone, which is not available in cinder. The control plane machine cannot be created. 
+
+[NOTE]
+====
+For the {product-title} clusters version 4.14 and higher, the installers validate the root AZ in `install-congif.yaml` file and ensures the deployment to be continued.
+====
```

Comment:
This idea could go after the first sentence.

'If you're upgrading, do this. If you're starting with 4.14, this is all done automatically.'

Reply:
Changed it to:- If you are starting with the {product-title} cluster, version 4.14, this process is done automatically. If you are  upgrading, follow the steps.